openslides/locale/tinymce/cs.js

```diff
@@ -0,0 +1,219 @@
+tinymce.addI18n('cs',{
```

Comment:
It's a copied file from http://archive.tinymce.com/i18n/index.php. TinyMCE core does not contains i18n files. And there is no bower package for translations to catch it more automatically. So we have to download new language files for tinymce manually and commit it to 'locale/tinymce'.